docs/usage/input.md

```diff
@@ -78,6 +78,18 @@ These options control how the directory space is traversed:
 - `--read-symlinks` Read files that are symbolic links (default: true)
 - `--mount` Don't descend directories on other filesystems (unix only)
 
+#### Testing
+
+You can test path and traversal filters using the `walk` command.
+
+For example:
+
+```sh
+rare walk -R --include '*txt' ./
+```
+
+Which will output matching paths.
```

Comment:
Note that this doesn't actually test the paths for readability or existence (eg. it will output what it's given)